docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/image/image_macros.twig

```diff
@@ -43,3 +43,29 @@
   </figure>
 
 {% endmacro %}
+
+{% macro ncidsImageHandler(image, caption, credit, crop) %}
```

Comment:
Ok, so this file is in the wrong place or is the wrong specificity. So in theory this macro could generically draw a "cgdp-image" in whatever container or usage. However, the image_style in use really depends on the caller. Additionally, the aspect ratio depends on the cgov_image display mode being used. So this logic is blurring cgov_image and cgov_home_landing concerns. I feel like you need to have this macro, call it `cgdpImageHandler` and the image passed in should either be the placeholder or the real image.

Reply:
Make the Crop be no_image_style